Dockerfile

```diff
@@ -5,8 +5,9 @@ ADD package.json package-lock.json /tmp/cirrus-ci-web/
 
 RUN npm ci
 
-ENV GENERATE_SOURCEMAP true
-ENV NODE_ENV production
+ENV \
```

Comment:
Why extra `\`?

Reply:
Because it makes it look more even

Reply:
It seems you'll still have the same indention. According to documentation you probably also want to use `=`: https://docs.docker.com/engine/reference/builder/#env

Plus in the example in the docs they even user single line syntax:

```Dockerfile
ENV NODE_ENV=production GENERATE_SOURCEMAP=true
```